spec/controllers/room_settings_controller_spec.rb

```diff
@@ -41,6 +41,7 @@
 
   describe '#update' do
     it 'uses MeetingOption::get_config_value and updates the setting if its config is "optional"' do
+      expect(MeetingOption).to receive(:get_config_value).and_call_original
```

Comment:
This is needed because `get_config_value` is called in the `after_create` method `create_meeting_option` in Room.